### PR TITLE
Feature/init

### PR DESCRIPTION
**Pull request by shmileee**

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+driver:
+  name: docker
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-ubuntu2004}-ansible:latest"
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
```
Comment (DamianBudelewski) [inline molecule/default/molecule.yml:13]: Should we add an example of port forwarding here? [Example](https://github.com/Appsilon/ansible-rstudio-workbench/blob/main/molecule/default/molecule.yml)

File: .yamllint [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+# Based on ansible-lint config
+
+extends: default
+
+rules:
```
Comment (DamianBudelewski) [inline .yamllint:6]: > When YAML is used as the format of a communication channel, it is useful to be able to indicate the end of a [document](https://yaml.org/spec/1.1/#document/syntax) without closing the [stream](https://yaml.org/spec/1.1/#stream/syntax), independent of starting the next [document](https://yaml.org/spec/1.1/#document/syntax). Lacking such a marker, the YAML [processor](https://yaml.org/spec/1.1/#processor/) reading the [stream](https://yaml.org/spec/1.1/#stream/syntax) would be forced to wait for the header of the next [document](https://yaml.org/spec/1.1/#document/syntax) (that may be long time in coming) in order to detect the end of the previous one. To support this scenario, a YAML [document](https://yaml.org/spec/1.1/#document/syntax) may be terminated by an explicit end line denoted by “...”, followed by optional [comments](https://yaml.org/spec/1.1/#comment/syntax). To ease the task of concatenating YAML [streams](https://yaml.org/spec/1.1/#stream/syntax), the end marker may be repeated.
[Source](https://yaml.org/spec/1.1/#document%20boundary%20marker/)

WDYT? Do we want to end our yamls like that or not?

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    rev: v0.15.1
+    hooks:
+      - id: gitlint
+        args: [--contrib=contrib-title-conventional-commits, --ignore=body-is-missing, --msg-filename]
+
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.28.1
+    hooks:
+      - id: markdownlint
+        exclude: .github/ISSUE_TEMPLATE
```
Comment (DamianBudelewski) [inline .pre-commit-config.yaml:26]: v0.31.1 is available. Is there anything that is blocking us from using it?

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+
+  - repo: https://github.com/jorisroovers/gitlint
+    rev: v0.15.1
+    hooks:
+      - id: gitlint
+        args: [--contrib=contrib-title-conventional-commits, --ignore=body-is-missing, --msg-filename]
```
Comment (DamianBudelewski) [inline .pre-commit-config.yaml:20]: v0.17.0 is available. Is there anything that is blocking us from using it?

File: README.md [hunk 5]
```diff
@@ -1 +1,125 @@
...
+```
+
+To perform quick test after some modification:
+
+```bash
+molecule create
+molecule converge
+molecule verify
+```
+
```
Comment (DamianBudelewski) [inline README.md:62]: Can we add molecule login here? I use quite often to test the running containers

**Review by DamianBudelewski — COMMENTED** [review]
Nice one! I added few small things, please check, we can discuss the end of yaml topic, update pre-commit modules and merge.

File: .pre-commit-config.yaml (hunk 3 above)
Comment (DamianBudelewski) [reply .pre-commit-config.yaml:26]: I wanted to ask if we can do that automatically :D That's much better!

File: molecule/default/molecule.yml (hunk 1 above)
Comment (shmileee) [reply molecule/default/molecule.yml:13]: this is very specific to ansible roles that deploy server-based applications, additionally we don't know the port beforehand

File: .yamllint (hunk 2 above)
Comment (shmileee) [reply .yamllint:6]: @DamianBudelewski what is the benefit for us in using this? Ansible playbooks do not support multiple YAML documents within the same file, so according to their docs the document-start/end is [optional](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics).

**Comment by shmileee** [comment]
> Nice one! I added few small things, please check, we can discuss the end of yaml topic, update pre-commit modules and merge.

I've pushed new changes and replied, PTAL.

File: molecule/default/molecule.yml (hunk 1 above)
Comment (DamianBudelewski) [reply molecule/default/molecule.yml:13]: Yup, either we can add that to readme or let someone find it in the docs when he needs to.

File: .yamllint (hunk 2 above)
Comment (DamianBudelewski) [reply .yamllint:6]: For me, it just looked better. We don't need to do it

**Pull request merged**